README.md

```diff
@@ -56,7 +56,7 @@ Once you setup the plugin correctly, you can call the `:generateSwagger` gradle
 
 In order to run this gradle plugin you need to fulfill the following requirements:
 
-* Gradle 5.x - This plugin uses Gradle 5 features, and you will need to setup your Gradle wrapper to use 5.4.1 or more.
+* Gradle 6.7+
```

Comment:
I'm not sure about the fact that to run the plugin you need Gradle6.7+

We have a test that ensures the min dependency and seems it is working properly
https://github.com/Yelp/swagger-gradle-codegen/blob/master/gradle-plugin/plugin/src/test/java/com/yelp/plugin/PluginTests.kt#L43

Considering this I would suggest to avoid changing this line or updating the test to reflect the new min version to use.

Reply:
You're correct. If the Gradle runner tests are passing, it's safe to assume is 5.4.1+ compatible.